app/lib/registration/sign_up_screen.dart

```diff
@@ -33,69 +38,90 @@ class SignUpScreen extends HookWidget {
+        ),
+        title: Text(
+          loc.signUpScreen_header,
+          style: const TextStyle(fontWeight: FontWeight.bold),
+        ),
         toolbarHeight: isPointer() ? 100 : null,
+        backgroundColor: backgroundColor,
       ),
+      backgroundColor: backgroundColor,
       body: SafeArea(
         minimum: EdgeInsets.only(
           bottom: isKeyboardShown ? Spacings.s : Spacings.l + Spacings.xxs,
         ),
-        child: Column(
-          children: [
-            Expanded(
-              child: LayoutBuilder(
-                builder: (context, constraints) {
-                  return SingleChildScrollView(
-                    keyboardDismissBehavior:
-                        ScrollViewKeyboardDismissBehavior.onDrag,
-                    padding: const EdgeInsets.only(
-                      left: Spacings.s,
-                      right: Spacings.s,
-                      bottom: Spacings.xl,
-                    ),
-                    child: ConstrainedBox(
-                      constraints: BoxConstraints(
-                        minHeight: constraints.maxHeight,
-                      ),
-                      child: Align(
-                        alignment: Alignment.topCenter,
-                        child: _Form(
-                          formKey: formKey,
-                          showErrors: showErrors.value,
+        child: Container(
+          color: colors.backgroundBase.secondary,
+          child: ConstrainedWidth(
+            child: Column(
+              children: [
+                Expanded(
+                  child: LayoutBuilder(
+                    builder: (context, constraints) {
+                      return SingleChildScrollView(
+                        keyboardDismissBehavior:
+                            ScrollViewKeyboardDismissBehavior.onDrag,
+                        padding: const EdgeInsets.only(
+                          left: Spacings.s,
+                          right: Spacings.s,
+                          bottom: Spacings.xl,
                         ),
-                      ),
-                    ),
-                  );
-                },
-              ),
-            ),
-            Container(
-              padding: const EdgeInsets.symmetric(horizontal: Spacings.m),
-              width: isSmallScreen(context) ? double.infinity : null,
-              child: _SignUpButton(formKey: formKey, showErrors: showErrors),
+                        child: ConstrainedBox(
+                          constraints: BoxConstraints(
+                            minHeight: constraints.maxHeight,
+                          ),
+                          child: Align(
+                            alignment: Alignment.topCenter,
+                            child: _Form(
+                              formKey: formKey,
+                              showErrors: showErrors.value,
+                              showAvatar: !isKeyboardShown,
```

Comment:
Is it really in our Designs, that the avatar is hidden, when they keyboard is shown? 


Again, there is a problem on Android with overlapping keyboard:

<img width="1280" height="2856" alt="Image" src="https://github.com/user-attachments/assets/433c36fd-2e84-4d73-bd04-4a934902b728" />